From 02cb9d6b467bb5696af4982a6d7df98e5cc2d3b7 Mon Sep 17 00:00:00 2001
From: Tsuyoshi Ozawa <ozawa@apache.org>
Date: Thu, 26 Nov 2015 01:02:27 +0900
Subject: [PATCH 2361/2848] CDH-50004 MAPREDUCE-6555. TestMRAppMaster fails on
 trunk. (Junping Du via ozawa)

(cherry picked from commit 3e85542a7afb8bbbc19f2aa59da04ec824168c0e)

Change-Id: I989a797e048e4ce66a3f8f4eeccbfc0c804dca7f
---
 .../hadoop/mapreduce/v2/app/TestMRAppMaster.java   |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/test/java/org/apache/hadoop/mapreduce/v2/app/TestMRAppMaster.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/test/java/org/apache/hadoop/mapreduce/v2/app/TestMRAppMaster.java
index d706af6..43267d5 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/test/java/org/apache/hadoop/mapreduce/v2/app/TestMRAppMaster.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/test/java/org/apache/hadoop/mapreduce/v2/app/TestMRAppMaster.java
@@ -145,6 +145,8 @@ public void testMRAppMasterMidLock() throws IOException,
     String userName = "TestAppMasterUser";
     JobConf conf = new JobConf();
     conf.set(MRJobConfig.MR_AM_STAGING_DIR, stagingDir);
+    conf.setInt(org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.
+        FILEOUTPUTCOMMITTER_ALGORITHM_VERSION, 1);
     ApplicationAttemptId applicationAttemptId = ConverterUtils
         .toApplicationAttemptId(applicationAttemptIdStr);
     JobId jobId =  TypeConverter.toYarn(
-- 
1.7.9.5

